app/views/layouts/application.html.slim

```diff
@@ -20,7 +20,10 @@ html
 
       - if flash.any?
         - flash.each do |type, message|
-          = content_tag :div, message, class: "flash-#{type}"
+         div class="flash-#{type}"
+             = message
+             span class="close-flash" 
+                 i class="fa fa-times-circle" aria-hidden="true"
```

Comment:
Maybe this icon could be done as a `:before` on the `close-flash` span? I'm a fan of keeping style elements like this in the stylesheets, what's your opinion on it? This could be done by adding 

```sass
=close-flash
  content: '\f057'
  font-family: 'FontAwesome'
```
to `stylesheets/base/_icons.sass`

and adding to the sass in `_flash.sass`:

```sass
   .close-flash
      margin-left: 0.5em
      cursor: pointer

      &:before
        +close
```

And also delete the icon line in the homepage markup:
```slim
/ in app/views/layouts/application.html.slim

 i class="fa fa-times-circle" aria-hidden="true"
````